<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditActionBase</class>
 <widget class="QWidget" name="EditActionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Action</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="KTabWidget" name="tabWidget3">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Function</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="theUseProfile">
         <property name="text">
          <string>&amp;Use application:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="theUseProfileAppLabel">
            <property name="text">
             <string>&amp;Application:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>theApplications</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="theApplications">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="thePerformFunction">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Perform function:</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="theFunctions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="theJustStart">
            <property name="text">
             <string>Just start application</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="theUseDBus">
         <property name="text">
          <string>Use &amp;D-Bus:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="theDBusApplicationsLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>App&amp;lication:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>theDBusApplications</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="theDBusApplications">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="theDBusObjectsLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>O&amp;bject:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>theDBusObjects</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="theDBusObjects">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="theDBusFunctionsLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fu&amp;nction:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>theDBusFunctions</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KComboBox" name="theDBusFunctions">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QTableView" name="argumentsView"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QRadioButton" name="theChangeMode">
           <property name="text">
            <string>Change &amp;mode to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="theModes">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="optionsDBusAppLabel">
         <property name="text">
          <string>&lt;b&gt;D-Bus/Application Action Options&lt;/b&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line2">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="theRepeat">
         <property name="text">
          <string>&amp;Action repeats if button is held down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="theAutoStart">
         <property name="text">
          <string>&amp;Start program/service if not already running</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer20">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>491</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="theMCOLabel">
         <property name="text">
          <string>&lt;b&gt;Mode Change Options&lt;/b&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line2_2">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="theDoBefore">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>E&amp;xecute all other actions before mode change (in original mode)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="theDoAfter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Exec&amp;ute all actions after mode change (in new mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer22">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>670</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="theIMTop">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Multiple Instances</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="theIMLine">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="theIMLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>This application may have multiple instances. In the case of having multiple instances at once when this action should be executed, choose a course of action:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="theDontSend">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Ignore the action</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="theSendToTop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send the action to the instance hi&amp;ghest in window stacking order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="theSendToBottom">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send the action to the instance &amp;lowest in window stacking order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="theSendToAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send the action to all i&amp;nstances</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget3</tabstop>
  <tabstop>theUseProfile</tabstop>
  <tabstop>thePerformFunction</tabstop>
  <tabstop>theApplications</tabstop>
  <tabstop>theJustStart</tabstop>
  <tabstop>theFunctions</tabstop>
  <tabstop>theUseDBus</tabstop>
  <tabstop>theDBusApplications</tabstop>
  <tabstop>theDBusObjects</tabstop>
  <tabstop>theRepeat</tabstop>
  <tabstop>theAutoStart</tabstop>
  <tabstop>theDBusFunctions</tabstop>
  <tabstop>theChangeMode</tabstop>
  <tabstop>theModes</tabstop>
  <tabstop>theDoBefore</tabstop>
  <tabstop>theDoAfter</tabstop>
  <tabstop>theDontSend</tabstop>
  <tabstop>theSendToTop</tabstop>
  <tabstop>theSendToBottom</tabstop>
  <tabstop>theSendToAll</tabstop>
 </tabstops>
 <includes>
  <include location="local">klineedit.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">keditlistbox.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
